tests/PipelineTest.php

```diff
@@ -66,7 +66,8 @@ public function testPipelineCreate(): void
 
         $result = $pipeline->create();
 
-        $this->assertContains('acknowledged', $result->getData());
+        $this->assertArrayHasKey('acknowledged', $result->getData());
```

Comment:
Are these changes related to PHP 8? The new phpunit version? This this new version also work in 7.x? 

This feeds into the same question like the php 7.3 part above: Could we have these change in a separate PR to make it clear they are not related to the PHP 8.0 changes?

Reply:
yes, so increase of php version should be merged before increasing phpunit version to 9.3 which supports php 8.0 but doesn't support php 7.2 anymore.

Reply:
It seems like it is also compatible with 7.3, so if we jump to 7.3, we can go to 9 first in a separate PR and do this changes?